PLAN.md

```diff
@@ -1,8 +1,8 @@
-- Syntax highlighting
-    - Fix regression / change in highlighting
-    - Support document symbol
-    - Only update symbol if the LS supports it
+- Switching files between workspace
+    - reason-of-life - good example
 
+- Semicolon issue? Why is there always a semicolon error, even though examples compile fine without it?
```

Comment:
This sounds like something isn't picking up `refmt` in your PATH. Could it be the case? How did you install merlin/reason/ocaml?